src/node_report.cc

```diff
@@ -61,8 +61,9 @@ static void WriteNodeReport(Isolate* isolate,
                             const std::string& filename,
                             std::ostream& out,
                             Local<Value> error,
-                            bool compact);
-static void PrintVersionInformation(JSONWriter* writer);
+                            bool compact,
+                            bool disable_network);
```

Comment:
If you pass a default value to this parameter, this change will not be a breaking change.